src/core/createConfig.js

```diff
@@ -63,24 +63,35 @@ const createConfig = config => {
     validate: () => {
       const keys = Object.keys(cfg.validators);
       const errors = keys.reduce((ac, key) => {
-        const validator = cfg.validators[key];
         const currentValue = cfg.get(key);
+        const validator = cfg.validators[key];
         if (
           currentValue == null &&
           Object.prototype.hasOwnProperty.call(validator, "defaultValue")
         ) {
           cfg.set(key, validator.defaultValue);
         } else if (validator.validate) {
-          // The validate property could be expanded into an
-          const errorMessage = validator.validate(
-            cfg,
-            key,
-            currentValue,
-            validator.defaultValue
-          );
-          if (errorMessage) {
-            ac.push(errorMessage);
+          const validators = [];
+          if (Array.isArray(validator.validate)) {
```

Comment:
Following up on @jonsnyder's comment about removing the `required` validator, it seems like we won't need this array of validators for now. Let's take it out; and if we need to use more than 1 validators, I should definitely go the composition route.

Reply:
I'm keeping the support for stringing together validators. I think we have sufficient functionality to validate pretty much anything we need going forward.